### PR TITLE
Eric dev

### PR DESCRIPTION
**Pull request by Fluff-E**
This is a pull request.  This has:

A maven project that will run the sample project using our laserfiche credentials.
It also has the Jackson library added for turning our Json file into a java object.  Not sure how to use it, but it's definitely there.
The example JSON file as .JSON and .txt

This is a good starting point to try and figure out how to use Jackson to process a JSON file

### CODE REVIEW

File: FileProcessingSystem/src/com/group7/fileprocessingsystem/FileProcessingSystem.java [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+ * @author Tacitor
+ */
+public class FileProcessingSystem {
+    
+    public static void main(String[] args) {
+        System.out.println("Stating file processor...");
+        System.out.println("Eric's Branch");
+    }
+    
+}
```
Comment (Tacitor) [inline FileProcessingSystem/src/com/group7/fileprocessingsystem/FileProcessingSystem.java:19]: Should delete this entire project and just used the Maven one

File: FileProcessor/DownloadedFile.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ligula mi, posuere at tincidunt in, sodales facilisis odio. Donec consequat tincidunt orci, id pretium nunc malesuada non. Donec ultricies mauris id lobortis sagittis. Donec tincidunt massa non ipsum vestibulum, a auctor est molestie. Nullam rutrum lectus vel fermentum tincidunt. Vivamus id pulvinar lacus. In porta fringilla metus, quis ultrices erat tempor id. Fusce tellus mauris, dapibus et molestie sed, ornare ut velit. Vestibulum nunc nisi, tincidunt tristique mollis in, dignissim vel purus. Sed pellentesque porttitor mauris in rutrum. Ut vitae tristique tellus, id gravida sem. Aliquam eu lorem at est pretium eleifend. Nulla a interdum lacus, ut eleifend diam. Suspendisse tincidunt, nulla sed condimentum tincidunt, nisl ipsum dignissim lorem, a pellentesque dolor lacus pharetra lectus.
```
Comment (Tacitor) [inline FileProcessor/DownloadedFile.txt:1]: What is this Lorem ipsum for?

File: FileProcessor/pom.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>19</maven.compiler.source>
+        <maven.compiler.target>19</maven.compiler.target>
+        <exec.mainClass>com.mycompany.mavenproject1.Mavenproject1</exec.mainClass>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>com.laserfiche</groupId>
+            <artifactId>lf-repository-api-client</artifactId>
```
Comment (Tacitor) [inline FileProcessor/pom.xml:17]: When you get a chance Eric can you explain the way you have the LaserFiche API setup here?

File: FileProcessor/src/main/java/com/group7/FileProcessor/FileProcessorMain.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+import java.util.function.Consumer;
+
+/**
+ *
+ * @author keric
+ */
+public class FileProcessorMain {
+
+     public static void main(String[] args) {
+        String servicePrincipalKey = "z6N-YywpamJ5Dr227qKb";
```
Comment (Tacitor) [inline FileProcessor/src/main/java/com/group7/FileProcessor/FileProcessorMain.java:26]: Really need to get these keys out of here. Laserfiche wants us to keep these off the internet.

**Review by Tacitor — CHANGES_REQUESTED** [review]
Must looks pretty good. Small issue of needing to delete the old NB project. Huge issue of keeping the Laserfiche credentials a secret.

File: FileProcessingSystem/src/com/group7/fileprocessingsystem/FileProcessingSystem.java (hunk 1 above)
Comment (Fluff-E) [reply FileProcessingSystem/src/com/group7/fileprocessingsystem/FileProcessingSystem.java:19]: done. see new commit.

File: FileProcessor/DownloadedFile.txt (hunk 2 above)
Comment (Fluff-E) [reply FileProcessor/DownloadedFile.txt:1]: This is the file that was downloaded in the sample program using the laserfiche api.

File: FileProcessor/pom.xml (hunk 3 above)
Comment (Fluff-E) [reply FileProcessor/pom.xml:17]: It's the sample program.

File: FileProcessor/src/main/java/com/group7/FileProcessor/FileProcessorMain.java (hunk 4 above)
Comment (Fluff-E) [reply FileProcessor/src/main/java/com/group7/FileProcessor/FileProcessorMain.java:26]: I have resolute doubt that there is actually a security risk caused by leaking our credentials to the repo of random text files.  Would you give 150 students the keys to something you actually cared about?

**Comment by Fluff-E** [comment]
I don't know how to conclude this process, but I tried to address the issues.

**Review by Tacitor — APPROVED** [review]
Looks good Ready to merge

**Pull request merged**